demo-snippet.html

```diff
@@ -125,6 +149,37 @@
 
         // Stamp the template.
         Polymer.dom(this).appendChild(document.importNode(template.content, true));
+      },
+
+      detached: function() {
+        this.unlisten(this.$.copyButton, 'mouseleave', '_resetCopyButtonState');
+        this.unlisten(this.$.copyButton, 'blur', '_resetCopyButtonState');
+      },
+
+      _copyToClipboard: function() {
+        // From https://github.com/google/material-design-lite/blob/master/docs/_assets/snippets.js
+        'use strict';
```

Comment:
Would you consider just adding `use strict` to the top of the script? We should probably be doing that everywhere anyway..